frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -49,21 +49,36 @@ const makeOCSRequest = (
 };
 
 export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>((props) => {
-  const {
-    handlePromise,
-    errorMessage,
-    inProgress,
-    match: {
-      params: { appName, ns },
-    },
-  } = props;
+  const { handlePromise, errorMessage, inProgress, match, setIsNewSCToBeCreated } = props;
+  const { appName, ns } = match.params;
   const [filteredNodes, setFilteredNodes] = React.useState<string[]>([]);
   const [storageClass, setStorageClass] = React.useState<StorageClassResourceKind>(null);
   const [nodes, setNodes] = React.useState<NodeKind[]>([]);
   // LVS: Local Volume Set
   const dispatch = useDispatch();
   const [LVSData, LVSLoaded, LVSLoadError] = useK8sWatchResource<K8sResourceKind[]>(LVSResource);
 
+  // const [scData, scLoaded, scLoadError] = useK8sWatchResource<StorageClassResourceKind[]>(
+  //   scResource,
+  // );
+
+  // React.useEffect(() => {
+  //   if ((scLoadError || scData.length === 0) && scLoaded) {
+  //     setIsNoProvSCPresent(false);
+  //   } else if (scLoaded) {
+  //     const filteredSCData = scData.filter(
+  //       (sc: StorageClassResourceKind) => sc?.provisioner === NO_PROVISIONER,
+  //     );
+  //     if (filteredSCData.length) {
+  //       setIsNoProvSCPresent(true);
+  //     }
+  //   }
+
+  //   return () => {
+  //     stopWatchK8sResource(scResource);
```

Comment:
Need to figure out a way to stop watching resource here as `stopWatchK8sResource` is used for class based component.